File: python/8 kyu/228.py
# https://www.codewars.com/kata/5b077ebdaf15be5c7f000077/train/python

# Passed

def count_sheep(n):
    sheep_sequence = [f'{sheep_count} sheep...' for sheep_count in range(1, n + 1)]
    sheep_sequence_str = ''.join(sheep_sequence)
    return sheep_sequence_str

output = count_sheep(3)
print(output)

File: nodejs/8 kyu/228.js
// https://www.codewars.com/kata/5966e33c4e686b508700002d/train/javascript

// Passed

function sumStr(a, b) {
  let result = Number(a) + Number(b);
  let resultStr = String(result);
  return resultStr;
}

const output = sumStr("34", "5");
console.log(output);